web/blog/2022-06-01-gitpod-hackathon-guide.md

```diff
@@ -0,0 +1,77 @@
+---
+title: How to win a hackathon. Brief manual.
+authors: [maksym36ua]
+tags: [wasp, gitpod, hackathon]
```

Comment:
we have no title image? shall we add something, pennywise one maybe?

Reply:
I'll come up with a pretty photo when I'll be able to find the initial image I'm using right now :)